index.js

```diff
@@ -32,7 +32,7 @@ class CircuitBreaker extends EventEmitter {
 
         this.command = command;
         this.breakerOptions = {
-            timeout: breakerOptions.timeout || 10000,
+            timeout: breakerOptions.timeout || 15000,
```

Comment:
can we configure `scm-github` to pass this value ?

Reply:
Yes, can do something like 
```
"scms": {
    "github": {
        "plugin": "github",
        "config": {
            "oauthClientId": "",
            "oauthClientSecret": "",
            "username": "sd-buildbot",
            "email": "dev-null@screwdriver.cd",
            "commentUserToken": "",
            "secret": "",
            "privateRepo": "false"
            "fusebox": {
                "breaker": {
                    "timeout": "15000"
                }
            }
        }
    }
}
```